### ISSUE TITLE
Error 0x8007001f after updating to 15048.rs2_release 

### ISSUE THREAD

**Issue by kyl191** [Microsoft/BashOnWindows#1767]
After updating my Windows 10 install to build 15048.rs2_release, Bash has stopped working with the error

```
Error: 0x8007001f
Press any key to continue...
```

* Expected results
Bash prompt appears as it used to

* Actual results (with terminal output if applicable)
Console just prints 
```
Error: 0x8007001f
Press any key to continue...
```

* Your Windows build number
Microsoft Windows [Version 10.0.15048]

I found #707, and ran the `lxcore_trace` files. Results are here: [lxcore.etl.zip.zip](https://github.com/Microsoft/BashOnWindows/files/831049/lxcore.etl.zip.zip)

It looks like an issue finding `/mnt`. I'm only running Windows Defender, and my computer is not part of an AD.

**Comment by kyl191** [comment]
Update: Rolled back to an earlier version, bash worked fine; upgraded to 15048 and bash doesn't work again.

Tried changing the `HKLM\SYSTEM\CurrentControlSet\Control\Session Manager\kernel\ dword:ObCaseInsensitive` value from 1 to 0, it had no effect.

**Comment by therealkenc** [comment]
`0x8007001f` is `ERROR_GEN_FAILURE` ("May be used to indicate that the device has stopped responding (hung) or a general failure has occurred on the device. The device may need to be manually reset"). That's awfully generic, so it could be #707 or could be anything.

I have been getting `0x80070005 ERROR_ACCESS_DENIED` on and off, but never post an issue because I have no good repro and rebooting always fixes. It would be nice if `bash.exe` had some better built-in diagnostics, because mere mortals aren't going to know how to run `lxcore_trace`, and the lazy (like me) never get around to it. It would be helpful if `bash.exe` dumped a stack backtrace on hard fails like this so people could easily post screen shots.

**Comment by sunilmut** [comment]
@kyl191 - Thanks for your post. Do you which build it was that you rolled back to and it worked? I looked at the traces and there is not much that has changed in this space recently. So, not sure what is going on. Can you try deleting your `%LocalAppData%\lxss\mnt` directory and restart the service to see if it makes any difference?

@therealkenc - That's a great idea, but there are also some challenges in getting there (not suggesting that it's not doable). There are many disparate entities in the subsystem (bash.exe, lxssmanager service, lxcore.sys) and there are multiple points of failure (in this case, it is the lxcore driver). Bubbling all that up to bash.exe requires some work. But, of course there are ways to achieve this. Such as, bubble up the backtrace from bash.exe, and perhaps use ETW for the service (driver already uses ETW). And, wrap it all up in a nice diagnostic tool (or within bash.exe itself when it fails to launch). Now, who will do the work? :). Maybe a user voice page will help :).

**Comment by kyl191** [comment]
@sunilmut I updated from build 14986 (the last slow build).

I renamed the `lxss\mnt` folder to `mnt-old` and ran bash. I now have a console, and the `mnt` folder was recreated.

I checked the permissions on `mnt-old`, and it only lists my user, not SYSTEM or Administrators. It apparently has inheritance disabled as well.

**Comment by sunilmut** [comment]
Good to know that you are up and running again. Adding @SvenGroot to see if the permission thing makes any sense to him? Not sure how your folder ended up in that state.

**Comment by therealkenc** [comment]
@sunilmut - Yeah I actually should have thought about it harder before I made the suggestion, and knew better. The a backtrace of `bash.exe` tells you "I had a problem with LxssManager", which well, we know. :)

As for User Voice, don't get me started... But keep up the great work and appreciate the feedback.

**Comment by benhillis** [comment]
@kyl191 - Have you had a chance to try this on Creators Update?  You may be hitting the same issue as #1862.

**Comment by cyberf3ead** [comment]
I installed Windows Creator Update and received Error: 0x8007001f when I attempted to start bash. I fixed it by ensuring SYSTEM was given permissions to each volume. When bash tries to mount your drives, if SYSTEM does not have permission, it will fail. Right click each volume, select properties, select the security tab, select advanced, select add, select a principal, type SYSTEM, select check names, select ok, select full control, select OK.
